modules/storage/src/androidTest/kotlin/edu/stanford/spezi/modules/storage/key/EncryptedSharedPreferencesKeyValueStorageTest.kt

```diff
@@ -0,0 +1,171 @@
+package edu.stanford.spezi.modules.storage.key
```

Comment:
Exactly, I had exactly the same problem with the missing application context that you described. I also came across Robolectric, but initially decided to check the entire functionality with an Android test. At that point, I also thought it made sense to let the user configure as little as possible in the part where encryption is important to avoid possible errors. However, with the factory method you mentioned in another comment, we could work around this issue, so it probably make sense to customise the implementation based on your suggestion. 👍